tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -4954,3 +4954,245 @@ def get_complex128(a):
 
     assert f_fl64_output == test_float64_output
     assert type(f_fl64_output) == type(test_float64_output.item())
+
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran", marks = [pytest.mark.fortran]),
+        pytest.param("c", marks = [
+            pytest.mark.skip(reason="Mod function not supported in C"),
+            pytest.mark.c]
```

Comment:
is there any problem with C ? I understand that for array it seem more complicated but for scalar type i think its need only a printer. is there any raison why you skip C tests ?

Reply:
the reason: array it seems more complicated, a printer for scalars is a good idea but is it worth implementing it without arrays?

Reply:
If it is trivial for scalars, and arrays are implemented element-wise then it is also trivial for arrays (see your solution for real/imag)

Reply:
you are right. about the tests i cannot do:
```python
s = shape(a)
return len(s), s[0], a[0]
```
because tuples are not implemented in C yet, So i think it is better to add missing C printers and activate their tests in another PR, since this issue/PR are just about Fortran

Reply:
Ok. Could you open an issue for `NumpyMod` in C please